test/Integration/SqlTriggerBindingIntegrationTests.cs

```diff
@@ -656,6 +656,72 @@ public async void LastAccessTimeColumn_Created_OnStartup()
             Assert.True(1 == (int)this.ExecuteScalar("SELECT 1 FROM sys.columns WHERE Name = N'LastAccessTime' AND Object_ID = Object_ID(N'[az_func].[GlobalState]')"), $"{GlobalStateTableName} should have {LastAccessTimeColumnName} column after restarting the listener.");
         }
 
+        /// <summary>
+        /// Tests that trigger function executes on table whose name is a reserved word (User).
+        /// </summary>
+        [Theory]
+        [SqlInlineData()]
+        [UnsupportedLanguages(SupportedLanguages.Java)] // test timing out for Java
```

Comment:
Please open an issue to track fixing this

Reply:
#883